### ISSUE TITLE
Spurious Unauthorized Error

### ISSUE THREAD

**Issue by dev-zero** [juruen/rmapi#45]
After one day has passed between two consecutive logins (not sure whether it is related, though), I get the following error when connecting:

```
ERROR: 2019/01/22 22:42:48 api.go:34: failed to fetch documents 401 Unauthorized Error
ERROR: 2019/01/22 22:42:48 main.go:30: retrying...
ReMarkable Cloud API Shell
```

I also saw the same error the first time I connected when I had to enter the token (on 2 different machines).

this is with version 0.0.4 (07cf6cb4a9cf8eed2d2db2e544146439ba80fbf7) with go 1.11.4

**Comment by ddvk** [comment]
imho this is by design, the tablet works like that. the access token expires and you need to get a new token

**Comment by dev-zero** [comment]
well, ok, but for a user of the CLI this is an implementation detail of the communication protocol

**Comment by ddvk** [comment]
fixed with #104
